src/compile/codegen.rs

```diff
@@ -817,12 +817,32 @@ fn mavm_codegen_expr<'a>(
 ) -> Result<(LabelGenerator, &'a mut Vec<Instruction>, usize), CodegenError> {
     let debug = expr.debug_info;
     let loc = expr.debug_info.location;
+
+    macro_rules! create {
+        ($opcode:ident) => {
+            Instruction::from_opcode(Opcode::AVMOpcode(AVMOpcode::$opcode), debug)
+        };
+        ($opcode:ident, $immediate:expr) => {
+            Instruction::from_opcode_imm(Opcode::AVMOpcode(AVMOpcode::$opcode), $immediate, debug)
+        };
+    }
+
     match &expr.kind {
         TypeCheckedExprKind::NewBuffer => {
             let opcode = Opcode::AVMOpcode(AVMOpcode::NewBuffer);
             code.push(Instruction::new(opcode, None, debug));
             Ok((label_gen, code, num_locals))
         }
+        TypeCheckedExprKind::Quote(bytes) => {
+            code.push(create!(
+                Noop,
+                Value::new_tuple(vec![
+                    Value::Int(Uint256::from_usize(2 * bytes.len())),
```

Comment:
Why do we multiply bytes.len by 2 here?

Reply:
Oops, that's wrong. I've committed a fix

Reply:
Thanks.  Looks good.